vue-frontend/src/App.vue

```diff
@@ -200,6 +217,31 @@ a {
   ) !important;
 }
 
+.v2-button {
+  border-radius: 3rem;
+  padding: 0.7rem 1.5rem;
+  text-align: center;
+  border: 1px solid #3d3d3d;
+  background-color: #fff;
+  color: #3d3d3d !important;
```

Comment:
Do we need `important` ?

Reply:
Yes because at hover time add default blue color